packages/datadog-esbuild/src/utils.js

```diff
@@ -0,0 +1,195 @@
+'use strict'
+
+// Based in import-in-the-middle
```

Comment:
What should we write here to indicated that the code is copied and modified from iitm? as IITM is not datadog property anymore I think we should mention it here.

Reply:
I usually write something along the lines of what you did here. Maybe a bit more format, eg:

```
// The content of this file is copied from the `import-in-the-middle` package with minor modifications (https://www.npmjs.com/package/import-in-the-middle)
```

Reply:
Could we somehow use IITM directly? We could potentially add a feature there?

I ask, since I fear it might at some point deviate and we do not notice to update the code here accordingly.

Reply:
> Could we somehow use IITM directly?

I tried, but the methods that I wanted are private and I couldn't use the methods directly, this PR is using some iitm methods, but the things that are here aren't there.
Anyway, this is copy & modifying some things from iitm.

> We could potentially add a feature there?

tbh, I am far to be confident enough to open a PR in IITM to properly support esbuild

> I ask, since I fear it might at some point deviate and we do not notice to update the code here accordingly.

This definitely could happen in the future, ideally we will totally move to orchestrionjs before it happens.